runtime/src/main/java/org/corfudb/runtime/object/ColumnFamilyRegistry.java

```diff
@@ -0,0 +1,13 @@
+package org.corfudb.runtime.object;
+
+import org.rocksdb.ColumnFamilyHandle;
+
+/**
+ * Provides ColumnFamilyHandles.
+ */
+public interface ColumnFamilyRegistry {
+
+    ColumnFamilyHandle getDefaultColumnFamily();
+
+    ColumnFamilyHandle getSecondaryIndexColumnFamily();
```

Comment:
I think encoding an Index ID as prefix is almost the same as using multiple column families provided by RocksDB (one for each Index). Is there any performance reasons why you choose to encode the index id by yourself?

Reply:
We (Zach) originally went down this path, and I decided to the current approach would:
* Be simpler, since we no longer have to maintain a secondary index registry.
* Reduce resource utilization, as every column family requires a separate memtable, block cache, bloom filters, etc.
* Performance would be more predictable, since data-set size would be comparable to the primary column family. When a column family data-set size is small, we generally never utilize  LSM `L1 - LN`. Searching just over `L0` tends to be unpredictable.